packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -205,6 +207,10 @@ const destructVmsPattern = pattern =>
         vms: destructPattern(pattern),
       }
 
+const isRetentionHigh = (settings, retention) =>
+  settings.getIn(['', 'fullInterval']) > RETENTION_LIMIT &&
```

Comment:
If `fullInterval` is `undefined`, this should return `true` but it will return `false`.